exp/api/v1alpha3/awsmachinepool_types.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha3
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	infrav1 "sigs.k8s.io/cluster-api-provider-aws/api/v1alpha3"
+	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	"sigs.k8s.io/cluster-api/errors"
+)
+
+const (
+	MachinePoolFinalizer        = "awsmachinepool.infrastructure.cluster.x-k8s.io"
+	LaunchTemplateLatestVersion = "$Latest"
+)
+
+// AWSMachinePoolSpec defines the desired state of AWSMachinePool
+type AWSMachinePoolSpec struct {
+	ProviderID           string            `json:"providerID,omitempty"` //TODO: is this needed?
+	AutoScalingGroupName string            `json:"autoScalingGroupName,omitempty"`
+	MinSize              int32             `json:"minSize,omitempty"`
+	MaxSize              int32             `json:"maxSize,omitempty"`
+	DesiredCapacity      int32             `json:"desiredCapacity,omitempty"`
```

Comment:
I don't think this is used here, since the controller respects `MachinePool.Spec.Replicas`, right?

Reply:
Actually this isn't used anywhere so I'll remove it